### PR TITLE
Development environment for firebase.

### PR DESCRIPTION
**Pull request by AO16**
@jimmay5469 I tried to move the firebase app and the github app to you. Also, you may need to update your deployment call to `ember serve --environment production` as `--environment development` is default.

### CODE REVIEW

**Comment by jimmay5469** [comment]
https://github.com/tonycoco/heroku-buildpack-ember-cli#heroku-buildpack-for-ember-cli-applications is what we're using and it says it generates a production build so I think this should be all we need.

File: config/environment.js [hunk 1]
```diff
@@ -48,7 +40,7 @@ module.exports = function(environment) {
   }
 
   if (environment === 'production') {
-
+    ENV.firebase = 'https://embernati-lightning.firebaseio.com/'
```
Comment (jimmay5469) [inline config/environment.js]: Semicolon.

File: config/environment.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = function(environment) {
       'script-src': "'self' https://*.firebaseio.com",
       'style-src': "'self' http://fonts.googleapis.com"
     },
-    firebase: 'https://embernati-lightning.firebaseio.com/',
+    firebase: 'https://dev-ember-lightning.firebaseio.com/',
```
Comment (jimmay5469) [inline config/environment.js]: Is this just a sample one you created on your account?  That should work fine if so, was just curious.

**Comment by jimmay5469** [comment]
LGTM otherwise, let me know when you make the update and I'll get it merged!

File: config/environment.js (hunk 2 above)
Comment (AO16) [inline config/environment.js]: I actually created this.

**Comment by AO16** [comment]
@jimmay5469, was trying to figure out what LGTM meant and the first thing that popped up was "let's get this money" haha

**Comment by jimmay5469** [comment]
LOL

**Pull request merged**